examples/light-switch-app/silabs/include/BindingHandler.h

```diff
@@ -29,5 +29,6 @@ struct BindingCommandData
     chip::EndpointId localEndpointId = 1;
     chip::CommandId commandId;
     chip::ClusterId clusterId;
-    bool isGroup = false;
+    bool isGroup         = false;
+    uint32_t args[7];
```

Comment:
This doesn't seem like the right wat to go to pass data along. We should a Data structure which leverages unions for each command type.

Reply:
Added a struct for all the types